### DISCUSSION TITLE
Can someone check if this works

### DISCUSSION THREAD

**Discussion by MilkMaker23** [3kh0/ext-remover#645] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
it is the extension laucher bookmarklet "fixed" by chat gpt

**Comment by 3kh0** [comment, marked as answer]
It still works, ChatGPT is stupid

**Reply by RavenChill-Dev** [reply]
@3kh0 Wanna do somthing together, and I mean it is BIG. Neworks central and project inn (I'm co-owners of both) want you to join them. CHeck out their stuff! [https://docs.google.com/document/d/1GDBI_ZMMPpZ4wgbXCmpd1FAmo64F4yeFEtV7RTJaNVk/edit?usp=sharing](url) and project in is still In dev.  My doc: [https://docs.google.com/document/d/1nIDPWnvNaM1K4zXV-yN16zxUNEr1JsH8YlZVLkaz1wI/edit?usp=sharing](url)

**Reply by ILikeBypassing** [reply]
dm him on discord bud

**Reply by RavenChill-Dev** [reply]
K
